CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+**Thanks for taking the time to contribute! ❤️**
+
+All types of contributions are encouraged and valued, no matter if it's a bug
+report 🐛, a feature request 💡, or a Pull Request 🚀.
+
+- **❓ I have a question:** Ask in [our Slack channel]
+- **🐛 I found a bug:** [Open an Issue]
+- **💡 I have an idea:** [Open an Issue]
+- **💻 I want to code:** See below
+
+If you like the project, but just don't have time to contribute, that's OK too!
+You can also star the project ⭐, rave about it online 💬, or add a link to us
+🔗 in your project's readme.
+
+⚠️ You must never report security 🔒 related issues, vulnerabilities or bugs
+including sensitive information to the issue tracker, or elsewhere in public.
+Instead, please [report them to the Microsoft Security Response Center]. You can
+read more about our security policy in [`SECURITY.md`].
+
+## Contributing code
+
+👨‍💼 This repository is a collection of dev container Features managed by dev
+container spec maintainers. Even though **we do not accept new Features in this
+repository**, we welcome contributions to [our existing Features]!
+
+🚀 If you're looking to create a new Feature, we encourage you to [create your
+own Feature repository] and add it to the index in [`collection-index.yml`].
```

Comment:
```suggestion
🚀 If you're looking to create a new Feature, then we encourage adding it to a separate repository of your control. The [devcontainers/feature-starter](https://github.com/devcontainers/feature-starter) repository has more guidance on self authoring Features.

💡 Once you've self authored the Features and if you'd like to share them with the community, then we'd recommend [adding it to the index](https://github.com/devcontainers/feature-starter#adding-features-to-the-index).
```

Reply:
Fixed in https://github.com/jcbhmr/features/commit/1a50ace247e56b9186609e9d76bef24666295bac

Reply:
Fixed in https://github.com/jcbhmr/features/commit/b3f4b8b975087af2029bea3581dcb15747bbbb74